src/main/java/com/github/britter/junit/jupiter/testkit/TestExecutionStatistics.java

```diff
@@ -0,0 +1,26 @@
+package com.github.britter.junit.jupiter.testkit;
+
+public interface TestExecutionStatistics {
+
+    Statistics getTestStatistics();
+
+    Statistics getContainerStatistics();
+
+    interface Statistics {
+        // TODO does this belong here?
+        long getReportingEntryPublishedCount();
+
+        // TODO does this belong here?
+        long getDynamicTestRegisteredCount();
```

Comment:
I would be more inclined to have something like `getEventCount(Type type)` to reduce API pollution